es-ds-components/components/es-dropdown-select.vue

```diff
@@ -123,17 +126,19 @@ const hide = () => {
                     height="1.125rem"
                     :class="[
                         {
-                            'text-gray-500': disabled,
-                            'text-gray-900': !disabled && state !== false,
+                            'text-gray-500': isDisabled,
+                            'text-gray-900': !isDisabled && state !== false,
                         },
                     ]" />
             </template>
             <template #option="slotProps">
-                <span>{{ slotProps.option.label ? slotProps.option.label : slotProps.option }}</span>
+                <span class="option-label">{{
+                    slotProps.option.label ? slotProps.option.label : slotProps.option
+                }}</span>
                 <icon-check
                     v-if="isSelected(slotProps.option)"
                     height="1.5rem"
-                    class="text-gray-700" />
+                    class="text-gray-700 flex-shrink-0 ml-25" />
```

Comment:
Now that the text wrapping is more pronounced, it's probably worth reserving space (via right padding) for the checkmark in _all_ cases and doing a position absolute on the checkmark,  so that the text doesn't shift once the checkmark appears.

Before click:
<img width="569" height="222" alt="Screenshot 2025-09-04 at 10 53 32 AM" src="https://github.com/user-attachments/assets/b5b4bf3c-96e9-4f90-994d-0281bb3a4462" />

After click:
<img width="575" height="217" alt="Screenshot 2025-09-04 at 10 53 40 AM" src="https://github.com/user-attachments/assets/79f8adb5-43cf-46da-87aa-00e905e42df1" />